### PR TITLE
Remove custom components

### PR DESCRIPTION
**Pull request by lasfin**
Tried to fix this issue: https://github.com/danilowoz/react-content-loader/issues/46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,8 @@ your custom loaders
 
 ```jsx
 // import the component
-import ContentLoader, { Rect, Circle } from 'react-content-loader'
+import ContentLoader from 'react-content-loader'
+import  { Rect, Circle } from 'create-react-content-loader'
```
Comment (danilowoz) [inline README.md]: Could you please remove this? Because it doesn't exist anymore

File: stories/index.js [hunk 2]
```diff
@@ -15,11 +14,7 @@ const Container = props => <div style={defaultStyle}>{props.children}</div>
 
 const MyLoader = () => {
   return (
-    <ContentLoader height={140} speed={1} primaryColor={'#333'} secondaryColor={'#999'}>
-      <Circle x={195} y={30} radius={30} />
-      <Rect x={50} y={80} height={10} radius={5} width={300} />
-      <Rect x={75} y={100} height={10} radius={5} width={250} />
-    </ContentLoader>
+    <ContentLoader height={140} speed={1} primaryColor={'#333'} secondaryColor={'#999'}/>
```
Comment (danilowoz) [inline stories/index.js]: Please, insert a custom loader, such as:

```html
    <rect x="80" y="17" rx="4" ry="4" width="300" height="13" /> 
    <rect x="82" y="44" rx="3" ry="3" width="250" height="10" /> 
    <circle cx="35" cy="35" r="35" />
```

File: tests/index.js [hunk 3]
```diff
@@ -52,19 +49,6 @@ describe('<ContentLoader />:', () => {
       })
     })
 
-    describe('when type is custom', () => {
```
Comment (danilowoz) [inline tests/index.js]: You shouldn't remove this test, it ensures the customs loaders will work, but now we have to test using a different way:

1. Insert a custom loader, like I mention above;
2. Test if the children are rendered (probably you haven't to change a lot of code);

If you have any question about test, feel free to call me :)

Comment (lasfin) [reply tests/index.js]: Will be this test ok?
```
const wrapper = mount(
      <ContentLoader>
        <rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
        <rect x="82" y="44" rx="3" ry="3" width="250" height="10" />
        <circle cx="35" cy="35" r="35" />
      </ContentLoader>
  )

  it('should render custom element', () => {
      expect(wrapper.find('rect')).to.have.length(3)
      expect(wrapper.find('circle')).to.have.length(1)
  })
```

Comment (danilowoz) [reply tests/index.js]: Sounds good, I think it's ok :)

If you want to organize more, put the `describe('when type is custom', () => {})` within the `describe('Type props are used', () => {})`, you know? But it isn't a problem :)

File: tests/index.js [hunk 4]
```diff
@@ -52,19 +49,22 @@ describe('<ContentLoader />:', () => {
...
+            <rect x="82" y="44" rx="3" ry="3" width="250" height="10" />
+            <circle cx="35" cy="35" r="35" />
+          </ContentLoader>
       )
 
       it('should render custom element', () => {
-        expect(wrapper).to.have.descendants(Rect)
-        expect(wrapper).to.have.descendants(Circle)
+          expect(wrapper.find('rect')).to.have.length(3)
+          expect(wrapper.find('circle')).to.have.length(1)
```
Comment (danilowoz) [inline tests/index.js]: The indentation of code is wrong here and above
But again, it isn't a big problem :)

File: stories/index.js [hunk 5]
```diff
@@ -15,11 +14,11 @@ const Container = props => <div style={defaultStyle}>{props.children}</div>
...
-    <ContentLoader height={140} speed={1} primaryColor={'#333'} secondaryColor={'#999'}>
-      <Circle x={195} y={30} radius={30} />
-      <Rect x={50} y={80} height={10} radius={5} width={300} />
-      <Rect x={75} y={100} height={10} radius={5} width={250} />
-    </ContentLoader>
+      <ContentLoader height={140} speed={1} primaryColor={'#333'} secondaryColor={'#999'}>
+          <rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
+          <rect x="82" y="44" rx="3" ry="3" width="250" height="10" />
+          <circle cx="35" cy="35" r="35" />
+      </ContentLoader>
```
Comment (danilowoz) [inline stories/index.js]: The indentation of code is wrong here
But it isn't a big problem :)

**Pull request merged**